pontoon/base/static/js/translate.js

```diff
@@ -1926,12 +1936,22 @@ var Pontoon = (function (my) {
           return;
         }
 
-        var source = $(this).find('.translation-clipboard').text();
+        // FTL Editor
+        if ($('#ftl').is('.active')) {
+          Pontoon.fluent.renderEditorWithTranslation({
+            pk: $(this).data('id'),
+            string: this.string
+          });
 
-        $('#translation').val(source).focus();
-        self.moveCursorToBeginning();
-        self.updateCurrentTranslationLength();
-        self.updateInPlaceTranslation();
+        // Standard Editor
+        } else {
+          var source = $(this).find('.translation-clipboard').text();
+
+          $('#translation').val(source).focus();
```

Comment:
I see that this code pattern occurs few times, maybe it's a good idea to create a method for that kind of render?